src/components/StockTable/index.jsx

```diff
@@ -0,0 +1,118 @@
+import { stockInfo } from '../../shared/stockInfo';
+import StockSymbol from '../StockSymbol';
+
+import './styles.css';
+import {
+    Table,
+    Thead,
+    Tbody,
+    Tr,
+    Th,
+    Td,
+    useColorModeValue,
+    VStack,
+    Container,
+    Text,
+    Code,
+} from '@chakra-ui/react';
+import React from 'react';
+import { useStockSymbolList } from '../../hooks/useStockList';
+
+const StockTable = () => {
+    const stocks = stockInfo;
+    const { stockDetails } = useStockSymbolList();
+    console.log('Stocks Context: ', stockDetails);
```

Comment:
Lets not commit these random console logs

Reply:
Will delete after it's been merged, it's not random. I need it there to find out what the context API was passing. Still need to connect it by creating another new context.